### PR TITLE
Track the longest streak, report on exit

### PR DESCRIPTION
**Pull request by danielmorrison**
I want to see my longest streak on exit, this provides that.

Great game idea!

### CODE REVIEW

**Comment by pklopping** [comment]
You beat me to it. Good idea!

File: git-game [hunk 1]
```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/ruby
 
+longest_streak = streak = 0
+
 # -- Exit gracefully --
 Signal.trap("INT") {
+  puts "\nLongest streak: #{longest_streak}"
   print "bye "
```
Comment (pklopping) [inline git-game]: I think the "bye " can be removed

File: git-game [hunk 2]
```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/ruby
 
+longest_streak = streak = 0
+
 # -- Exit gracefully --
 Signal.trap("INT") {
+  puts "\nLongest streak: #{longest_streak}"
```
Comment (pklopping) [inline git-game:7]: I'd also put a `\n` at the end of this puts.

**Comment by danielmorrison** [comment]
Removed "bye". `puts` will add the trailing `\n` for us.

**Comment by jsomers** [comment]
@danielmorrison thanks! merged in https://github.com/jsomers/git-game/pull/26

**Pull request merged**